.pre-commit-config.yaml

```diff
@@ -79,6 +79,12 @@ repos:
       # GoSec
       - id: go-sec-repo-mod
 
+  # Nix hooks
+  - repo: https://github.com/nix-community/nixpkgs-fmt
+    rev: v1.3.0
+    hooks:
+      - id: nixpkgs-fmt
```

Comment:
I would like to see this used in a test branch based on https://github.com/cisagov/awssh/pull/27 to confirm both that files are being checked and that it works as expected.

Reply:
Great idea!  I can quickly set that up right now.

Reply:
@mcdonnnj - Please see:
- [The branch](https://github.com/cisagov/awssh/tree/test-skeleton-generic-change).
- [The relevant output from the relevant workflow run](https://github.com/cisagov/awssh/actions/runs/6098128334/job/16547298034#step:15:44).
- [The commit wherein I updated the Nix flake formatting](https://github.com/cisagov/awssh/pull/27/commits/5662fa2f4e4f8db8d70cff3eef6abe53659ea9a8) in cisagov/awssh#27.

Reply:
Thanks @jsf9k !